src/components/Card/PlaylistCard.js

```diff
@@ -78,7 +78,7 @@ export const PlaylistHeader = ({response, expanded}) => {
 }
 PlaylistHeader.propTypes = {
   meta: PropTypes.object,
-  expanded: PropTypes.oneOf(expansions)
+  expanded: PropTypes.oneOf([false, 'horizontal', 'vertical'])
```

Comment:
Is expansions the same as `[false, 'horizontal', 'vertical']`? If so, would be better to keep it as `expansions` to use the same source.